extensions/amp-analytics/amp-analytics.md

```diff
@@ -173,6 +173,87 @@ In this example, we specify the `config` attribute to load the configuration dat
 <amp-analytics config="https://example.com/analytics.account.config.json">
 ```
 
+#### Configuration Rewriter
+
+The configuration rewriter feature is designed to allow analytics providers to dynamically rewrite a provided configuration. This is similar to the remote configuration feature but additionally includes any user-provided configuration in the request made to the sever. This currently can only be enabled by an analytics vendor.
+
+An analytics vendor will specify a `configRewriter` property with a server url.
+```js
+export const VENDOR_ANALYTICS_CONFIG = {
+    ...
+    'configRewriter': {
+      'url': 'https://www.vendor.com/amp-config-rewriter',
+    },
+    ...
+}
+```
+
+The runtime will fetch the remote configuration (if given) and merge it with the configuration inlined in the document, before sending a request to `configRewriter` endpoint given by the vendor. The vendor can then use this data server side to construct and return a new rewritten configuration.
+
+The runtime will then merge all these configurations in order to determine the final configuration. This will happen in this order of precedence (highest to lowest):
```

Comment:
"The runtime will then merge all these configurations in order to determine the final configuration. This will happen in this order of precedence (highest to lowest):"

to

"The runtime then merges all the provided configuration to determine the final configuration in order of highest to lowest precedence:"

Reply:
Done.